build.sbt

```
@@ -0,0 +1,26 @@
+import com.typesafe.sbt.packager.docker.Cmd
+
+name := "codacy-metrics-detekt"
+
+scalaVersion := "2.13.3"
+
+val detektVersion = "1.14.2"
+
+resolvers += Resolver.jcenterRepo
+
+scalacOptions in (Compile, console) ~= {
```

You can use the slash syntax here:
```suggestion
console / Compile / scalacOptions ~= {
```